sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -945,7 +805,7 @@ def _get_path_properties(self, **kwargs):
         :returns:
             Information including user-defined metadata, standard HTTP properties,
             and system properties for the file or directory.
-        :rtype: DirectoryProperties or FileProperties
+        :rtype: BlobProperties
```

Comment:
Did you verify this with breakpoints? Just curious because I originally assumed it would return a `PathProperties`, but the hint is `DirectoryProperties or FileProperties`, but you resolved it to `BlobProperties`

Reply:
I definitely agree with your thought process, but I think our docs were originally wrong :) 

Below on line 824, we make the call to

```python3
self._blob_client.get_blob_properties(**kwargs)
```

which by the API docs seem to return type `BlobProperties`.

Reply:
We definitely do not return a BlobProperties :)

If you look at the file or directory client who call this, they each pass a specific cls which converts the `BlobProperties` to a different properties.

Reply:
Sounds good, will make the change.